tests/operator/operator.go

```diff
@@ -242,19 +241,6 @@ var _ = Describe("[Serial][sig-operator]Operator", Serial, decorators.SigOperato
 
 		}
 
-		parseDaemonset = func(name string) (imagePrefix string) {
```

Comment:
Why is this func better inlined? (I don't know, I wish the commit message would tell me)

Reply:
```
As the waitForKv lambda only called once, this commit inlines its logic
and removes the function itself.
```
but why is this an improvement? the `It` context is very long and convoluted a it is, so I would not mind more encapsulation there, not less. But then again, I don't have a strong opinion.

Reply:
@dankenigsberg - done. could you PTAL?